Application/src/Pages/DashBoard/GuestDashboard.js

```diff
@@ -0,0 +1,107 @@
+import "./GuestDashboard.css";
+import React, { useState } from "react";
+import MagicDropZone from "react-magic-dropzone";
+import { ToastContainer, toast } from "react-toastify";
+import NavBar from "../../Components/NavBar";
+import BarGraph from "../../Components/BarGraph";
+import AccessibilityBarGraphData from "../../Components/AccessibilityBarGraphData";
+import ConvertImageToBase64 from "../../Utils/ConvertImageToBase64";
+import { getImageGrid } from "../../Utils/Structure";
+
+const GuestDashboard = () => {
+  const [filePreview, setFilePreview] = useState(null);
+  const [posterGrade, setPosterGrade] = useState({
+    textRating: 1,
+    structureRating: 1,
+    colorRating: 1,
+  });
+  const [totalCalculationTime, setTotalCalculationTime] = useState(0.0);
+  const [calculating, setCalculating] = useState(false);
+
+  async function getAccessibilityScore(poster) {
+    let startTime = Date.now();
+    setCalculating(true);
+    toast.info("Uploading image...", {
+      position: toast.POSITION.BOTTOM_RIGHT,
+      autoClose: 2000,
+    });
+    poster = await ConvertImageToBase64(poster);
+    toast.info("Calculating accessibility score...", {
+      position: toast.POSITION.BOTTOM_RIGHT,
+      autoClose: 4000,
+    });
+    await getImageGrid("data:image/png;base64," + poster)
+      .then((score) => {
+        setPosterGrade({
+          textRating: Math.round(score.textGrade),
+          structureRating: Math.round(score.structureGrade),
+          colorRating: Math.round(score.colorGrade),
+        });
+      })
+      .catch(() => {
+        let endTime = Date.now();
```

Comment:
Remove this statement because it is never used

Reply:
done